packages/core/content-manager/server/controllers/collection-types.js

```diff
@@ -329,6 +329,31 @@ module.exports = {
 
     const number = await entityManager.getNumberOfDraftRelations(id, model);
 
+    return {
+      data: number,
+    };
+  },
+  async getMultipleEntriesNumberOfDraftRelations(ctx) {
+    const { userAbility } = ctx.state;
+    const params = new URLSearchParams(ctx.querystring);
+    const ids = params.get('ids').split(',');
+    const { model } = ctx.params;
+
+    const entityManager = getService('entity-manager');
+    const permissionChecker = getService('permission-checker').create({ userAbility, model });
+
+    if (permissionChecker.cannot.read()) {
+      return ctx.forbidden();
+    }
+
+    const entities = await entityManager.find(ids, model);
+
+    if (!entities || entities.length !== ids.length) {
+      return ctx.notFound();
+    }
```

Comment:
you might want to update the branch with main, this find will need to receive a populate object, so all relations are populated with counts

```js
    const populate = await getService('populate-builder')(model)
      .populateDeep()
      .countRelations()
      .build();

   const entities = await entityManager.find(ids, model, { populate } );
```

This populate was done before in the entity manager, but because we tried to simplify it and we wanted to improve the permission system we moved thi slogic into the controllers.

I am in the process of documenting this 🚀

Reply:
thanks a lot for the suggestion @Marc-Roig , done